packages/frontend/src/components/ControlledAutocomplete/index.tsx

```diff
@@ -149,7 +164,10 @@ function ControlledAutocomplete(
                 ? {
```

Comment:
<!--__GRAPHITE_HTML_TAG_START__--><i class='graphite__hidden'>[Re: line 152]<br/><br/></i>

<!--__GRAPHITE_HTML_TAG_END__-->can we also add 
```
isReadOnly={isCreatingNewOption}
```<!--__GRAPHITE_HTML_TAG_START__--><span class='graphite__hidden'><br/><br/>See this comment inline on <a href="https://app.graphite.dev/github/pr/opengovsg/plumber/661?utm_source=unchanged-line-comment">Graphite</a>.</span>

<!--__GRAPHITE_HTML_TAG_END__-->